mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterScalerActor.java

```diff
@@ -138,32 +146,30 @@ public Receive createReceive() {
             ReceiveBuilder
                 .create()
                 .match(TriggerClusterUsageRequest.class, this::onTriggerClusterUsageRequest)
+                .match(TriggerClusterRuleRefreshRequest.class, this::onTriggerClusterRuleRefreshRequest)
+                .match(GetRuleSetRequest.class,
+                    req -> getSender().tell(
+                        GetRuleSetResponse.builder().rules(ImmutableMap.copyOf(this.skuToRuleMap)).build(), self()))
                 .match(GetClusterUsageResponse.class, this::onGetClusterUsageResponse)
                 .match(GetClusterIdleInstancesResponse.class, this::onGetClusterIdleInstancesResponse)
+                .match(GetRuleSetResponse.class,
+                    s -> log.info("[{}] Refreshed rule size: {}", s.getClusterID(), s.getRules().size()))
                 .match(Ack.class, ack -> log.info("Received ack from {}", sender()))
                 .build();
     }
 
     private void init() {
```

Comment:
can you move this to AbstractActor::preStart? https://docs.huihoo.com/akka/2.5/guide/tutorial_1.html

Reply:
This log is actually designed to be here to help verify the refreshed rules are correct. I was running into error when some deser error made and added this to help trace the root cause.